microservices/app/__init__.py

```diff
@@ -38,19 +31,34 @@ def create_app(config_name):
     Swagger(app, config=api_config)
 
     #  # HTTP error handling
-    #  @app.errorhandler(404)
-    #  def not_found(error):
-    #      return render_template('error/404.html'), 404
+    @app.errorhandler(404)
```

Comment:
@Abezzam10 This is a global exception handler any time there is an unhandled exception.